python/ray/serve/model_wrappers.py

```diff
@@ -1,27 +1,25 @@
-from typing import Dict, Optional, Type, Union
+from typing import Callable, Dict, List, Optional, Tuple, Type, Union
+import numpy as np
```

Comment:
Do we want to make this a delayed import at all? I don't think Serve currently relies on numpy.

Reply:
ray core actually depends on numpy! 🎉